x-pack/legacy/plugins/integrations_manager/common/routes.ts

```diff
@@ -18,3 +18,15 @@ export function getListPath() {
 export function getInfoPath(pkgkey: string) {
   return API_INFO_PATTERN.replace('{pkgkey}', pkgkey);
 }
+
+export function getInstallPath(pkgkey: string, asset: string = '') {
```

Comment:
Not used by this PR. Added for completeness (get* for each path). Copied from https://github.com/jfsiii/kibana/pull/2